blog/2024-02-27-extraction-matters-most/index.mdx

```diff
@@ -0,0 +1,135 @@
+---
+slug: extraction-matters-most
+title: Extraction Matters Most
+image: ./heatmap_extractor.png
+description: |
+  Experimental measurements of a variety of options in a RAG pipeline lead
+  to the conclusion that how you extract content from documents matters most.
+keywords:
+  - retrieval augmented generation
+  - RAG experimental evaluation
```

Comment:
I would paste the content into ChatGPT and have it generate a YAML-formatted SEO keyword list just to fill this out.